vortex-array/src/compute/conformance/cast.rs

```diff
@@ -111,7 +111,7 @@ fn test_cast_from_bool(array: &dyn Array, nullability: Nullability) {
     test_cast_nullability_changes(array, &DType::Bool(Nullability::Nullable));
     if nullability == Nullability::Nullable {
         // Try casting to non-nullable (may fail if nulls present)
-        let _ = cast(array, &DType::Bool(Nullability::NonNullable));
+        cast(array, &DType::Bool(Nullability::NonNullable)).vortex_unwrap();
```

Comment:
This file had several tests that did not actually run their tests because they didn't try to unwrap the result.